content/pages/meetup/index.rst

```diff
@@ -10,4 +10,10 @@ meetup 2023
 ==============
 
 * `Meetup 2023 March at NYCU
-  <{filename}2023/03-nycu.rst>`__
\ No newline at end of file
+  <{filename}2023/03-nycu.rst>`__
```

Comment:
Could you please use descending order like the sprint listing page: https://sciwork.dev/sprint/ ?